xdsl/dialects/riscv_scf.py

```diff
@@ -87,10 +87,16 @@ def verify_(self):
                     f"The first block argument of the body is of type {iter_var.type}"
                     " instead of riscv.IntRegisterType"
                 )
-        for arg, block_arg in zip(self.iter_args, self.body.block.args[1:]):
+        for idx, (arg, block_arg) in enumerate(
+            zip(
+                self.iter_args,
+                self.body.block.args[1:],
+                strict=True,
```

Comment:
This `strict` parameter is not required since the verifier checks earlier that `(len(self.iter_args) + 1) == len(self.body.block.args)`.
Also, while I don't mind too much the change here, could you please use next time the comment feature like Sasha suggested ?